classes/api/forms.php

```diff
@@ -370,6 +387,75 @@ public function id_endpoint_url() {
         return '/' . $this->route_base() . '/(?P<form_id>[\w-]+)';
     }
 
+    /**
+     * Create a form
+     *
+     * @since 1.8.0
+     *
+     * @param WP_REST_Request $request Full data about the request.
+     * @return Caldera_Forms_API_Response|Caldera_Forms_API_Error
+     */
+    public function create_item( WP_REST_Request $request ) {
+        $data = [];
+        foreach ( array_keys($this->args_for_create()) as $key ){
+            $data[$key] = $request[$key];
+        }
+
+        $new_form = Caldera_Forms_Forms::create_form( $data );
```

Comment:
Using `Caldera_Forms_Forms::import_form()` would provide consistent sanitization and validation. it would also duplicate sanitization/validation WP REST API is already doing.